hack/verify-gomod.sh

```diff
@@ -24,7 +24,7 @@ cd "${KOPS_ROOT}"
 
 make gomod
 
-changes=$(git status --porcelain || true)
+changes=$(git status --porcelain vendor/ || true)
```

Comment:
I think this might remove the assertion that go.mod/go.sum are tidy. It also removes the (redundant) bazel assertion.

Reply:
Good catch. Updated.